CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -58,6 +59,21 @@ local SEARCH_BACKGROUND_FADE = 0.15
 
 local DOUBLE_CLICK_TIME = 0.5
 
+local ZERO_KEY_VALUE = Enum.KeyCode.Zero.Value
```

Comment:
I suppose the original author of this code thought it would be a worthy optimization!